tests/DoctrineMetadataTest.php

```diff
@@ -0,0 +1,104 @@
+<?php
+
+declare(strict_types=1);
+
+namespace DotTest\DoctrineMetadata;
+
+use Dot\DoctrineMetadata\Factory\DoctrineMetadataMapFactory;
+use Mezzio\Hal\Metadata\Exception\InvalidConfigException;
+use Mezzio\Hal\Metadata\MetadataMap;
+use PHPUnit\Framework\TestCase;
+use Psr\Container\ContainerInterface;
+
+class DoctrineMetadataTest extends TestCase
+{
+    private DoctrineMetadataMapFactory $doctrineMetadataMapFactory;
+    private ContainerInterface $container;
+
+    public function setUp(): void
+    {
+        parent::setUp();
```

Comment:
You can remove the call to `parent::setUp();`, because it does nothing.
The `setUp` method in `TestCase` is there only to make sure that the method is callable before each test, even if a test does not implement it.